docs/docs/sample-scripts/spo/add-multiple-lists-in-multiple-sites.md

```diff
@@ -0,0 +1,68 @@
+# Add multiple lists in multiple sites
```

Comment:
Let's add a reference to the file in mkdocs.yml so that it's included in the generated docs